test/apify/enqueue_links/enqueue_links.test.ts

```diff
@@ -288,29 +338,11 @@ describe('enqueueLinks()', () => {
                 enqueued.push(request);
             };
 
-            await browserCrawlerEnqueueLinks({
+            await expect(async () => browserCrawlerEnqueueLinks({
```

Comment:
you can do just this when testing async methods with `expect().resolves/rejects`
```suggestion
            await expect(browserCrawlerEnqueueLinks({
```